include/SDL_vulkan.h

```diff
@@ -161,6 +161,18 @@ extern DECLSPEC SDL_bool SDLCALL SDL_Vulkan_GetInstanceExtensions(SDL_Window *wi
                                                                   unsigned int *pCount,
                                                                   const char **pNames);
 
+/**
+ * Get the device driver's string representation.
+ *
+ * \param id The id of the device
+ * \returns the device's name
+ *
+ * \since This function is available since SDL 2.0.23.
+ *
+ * \sa SDL_Vulkan_GetDriverVendorString
+ */
+extern DECLSPEC const char *SDLCALL SDL_Vulkan_GetDriverVendorString(uint32_t id);
```

Comment:
```suggestion
/**
 * Translates the VkPhysicalDevice's "vendorID" (for example, a PCI vendor ID) to a human-readable string.
 *
 * \param id The vendorID field of VkPhysicalDeviceProperties
 * \returns The vendor's human-readable name
 *
 * \since This function is available since SDL 2.0.23.
 *
 * \sa SDL_Vulkan_GetDriverVendorString
 */
extern DECLSPEC const char *SDLCALL SDL_Vulkan_GetDriverVendorString(uint32_t id);
```